docs/chrraces.md

```diff
@@ -0,0 +1,79 @@
+| 14     | unk_322           |              | faction (0 Alliance, 1 Horde, 2 not available?) |
+| 15-30  | name              | String + Loc | used for DBC language detection/selection       |
+| 31     |                   |              | string flags, unused                            |
+| 32-47  | nameFemale        | String + Loc | if different from base (male) case              |
+| 48     |                   |              | string flags, unused                            |
+| 49-64  |                   |              | if different from base (male) case              |
+| 65     |                   |              | string flags, unused                            |
+| 66-68  |                   |              | unused                                          |
+| 69     | expansion         | Integer      | (0 - original race, 1 - tbc addon, ...)         |
+
+### Content
+
+| ID | Value    | Name                    |
+|----|----------|-------------------------|
+| 1  | 1        | Human                   |
+| 2  | 2        | Orc                     |
+| 3  | 4        | Dwarf                   |
+| 4  | 8        | Night Elf               |
+| 5  | 16       | Undead                  |
+| 6  | 32       | Tauren                  |
+| 7  | 64       | Gnome                   |
+| 8  | 128      | Troll                   |
+| 9  | 256      | Goblin (4.x)            |
+| 10 | 512      | Blood Elf               |
+| 11 | 1024     | Draenei                 |
+| 12 | 2048     | Fel Orc                 |
+| 13 | 4096     | Naga                    |
+| 14 | 8192     | Broken                  |
+| 15 | 16384    | Skeleton                |
+| 16 | 32768    | Vrykul                  |
+| 17 | 65536    | Tuskarr                 |
+| 18 | 131072   | Forest Troll            |
+| 19 | 262144   | Taunka                  |
+| 20 | 524288   | Northrend Skeleton      |
+| 21 | 1048576  | Ice Troll               |
+| 22 | 2097152  | Worgen (4.x)            |
+| 24 | 8388608  | Pandaren Neutral (5.x)  |
+| 25 | 16777216 | Pandaren Alliance (5.x) |
+| 26 | 33554432 | Pandaren Horde (5.x)    |
+
+### How do I get the values?
+
+Through the powers of 2. It can be better understood in the following table:
+
+| N  | N - 1 | 2 exp (N - 1) | Races        |
+|----|-------|---------------|--------------|
+| 1  | 0     | 1             | Human        |
+| 2  | 1     | 2             | Orc          |
+| 3  | 2     | 4             | Dwarf        |
+| 4  | 3     | 8             | Night Elf    |
+| 5  | 4     | 16            | Undead       |
+| 6  | 5     | 32            | Tauren       |
+| 7  | 6     | 64            | Gnome        |
+| 8  | 7     | 128           | Troll        |
+| 9  | 8     | 256           | Goblin (4.x) |
+| 10 | 9     | 512           | Blood Elf    |
+| 11 | 10    | 1024          | Draenei      |
+
+![raceMask](https://user-images.githubusercontent.com/2810187/104961176-909b1500-59b4-11eb-8a55-2db891410ba7.png)
```

Comment:
I don't know if this is necessary, it is basic bit work and it is explained in another page https://www.azerothcore.org/wiki/Bit-and_bytes-tutorial

Reply:
> I don't know if this is necessary, it is basic bit work and it is explained in another page https://www.azerothcore.org/wiki/Bit-and_bytes-tutorial

I did not know about the existence of this file.
We could then create a link within the document.

Reply:
It has the b in capital letters xD.

Reply:
I know, sucks.

But I still think it is not necessary. Perhaps add something along the lines to "To use multiple races you add the bit values together` but nothing more. I don't think it is necessary to explain it :)

Reply:
I already added the link to the document you mentioned